### ISSUE TITLE
SingleLineAfterImportsFixer broken

### ISSUE THREAD

**Issue by GrahamCampbell** [FriendsOfPHP/PHP-CS-Fixer#1957]
Consider the following input file:

```php
<?php namespace Foo;

use Bar\Baz;

/**
 * Foo.
 */
```

We'll end up with:

```
RuntimeException: Index invalid or out of range in /root/PHP-CS-Fixer/src/Fixer/Import/SingleLineAfterImportsFixer.php:69
Stack trace:
#0 /root/PHP-CS-Fixer/src/Runner/Runner.php(189): PhpCsFixer\Fixer\Import\SingleLineAfterImportsFixer->fix(Object(Symfony\Component\Finder\SplFileInfo), Object(PhpCsFixer\Tokenizer\Tokens))
#1 /root/PHP-CS-Fixer/src/Runner/Runner.php(141): PhpCsFixer\Runner\Runner->fixFile(Object(Symfony\Component\Finder\SplFileInfo), Object(PhpCsFixer\Linter\LintingResult))
#2 /root/PHP-CS-Fixer/src/Console/Command/FixCommand.php(364): PhpCsFixer\Runner\Runner->fix()
#3 /root/PHP-CS-Fixer/vendor/symfony/console/Command/Command.php(256): PhpCsFixer\Console\Command\FixCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /root/PHP-CS-Fixer/vendor/symfony/console/Application.php(815): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /root/PHP-CS-Fixer/vendor/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(PhpCsFixer\Console\Command\FixCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /root/PHP-CS-Fixer/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /root/PHP-CS-Fixer/php-cs-fixer(45): Symfony\Component\Console\Application->run()
#8 {main}
```

**Comment by keradus** [comment]
It's awesome you reported more and more failing cases.
Please, remember to always provide min supported version in which the bug occurs.

Also, we won't be able to solve everything. Feel free to help with bugfixing.

**Comment by keradus** [comment]
Fixed by #1961, closing
